examples/tariff_14_step_size.json

```diff
@@ -50,7 +50,7 @@
       ],
       "restrictions" : {
         "start_time" : "20:00",
-        "end_time" : "24:00"
+        "end_time" : "23:59"
```

Comment:
Good point!! 24:00 is wrong. Even violates the regex. 
Going for 23:59 will not solve the problem, as you have written: What about the time between 23:59 and 00:00.
What about a time slot from 20:00 until 06:00 next morning, is that allowed?

I would say so. So for me, the best solution is: 00:00. But we also need to explain this better in 11.4.6

Reply:
How about something if end_time < start_time then wrap to next day?  Or can we specify start_date/time and end_date/time, so really start/end timestamps?  Which is a pain because we really want daily opening times .  But it's not impossible to have something that is open from 6:00a to 2:00a and only closed for 4 hours.